input/fsh/profiles/FinnishAppointmentPatient.fsh

```diff
@@ -0,0 +1,33 @@
+Profile: FinnishAppointmentPatient
```

Comment:
There's probably nothing scheduling specific here? We should change to use the FinnishPatient from the base profiles.